screens/SettingsScreen/SettingsScreen.tsx

```diff
@@ -0,0 +1,697 @@
+import { Caption, Headline, Title } from "@components/Text"
+import { Ionicon } from "@components/common/Icons"
+import { AppStyles } from "@lib/AppColorStyle"
+import { useNavigation } from "@react-navigation/native"
+import React, { useState } from "react"
+import { ScrollView, StyleSheet, Switch, View } from "react-native"
+import { Divider } from "react-native-elements"
+
+export type SwitchComponentProps = {
+  value: boolean
+  onValueChange: (value: boolean) => void
+}
+
+export const SettingsScreen = () => {
+  // List of attendees
+
+  const navigation = useNavigation()
+  /**
+   * Creates a switch for a given setting, and uses states to keep track of its current on-off status.
+   *
+   * @param setting: A given setting you want to keep track of.
+   * @returns A pre-built Switch component for this specific setting.
+   */
+
+  const SwitchComponent = ({ value, onValueChange }: SwitchComponentProps) => {
```

Comment:
Make the component organisations outside of the export const SettingsScreen, to avoid re-renders